docs/src/guides/drupal9/redis.md

```diff
@@ -1,149 +1,172 @@
-  $settings['bootstrap_container_definition'] = [
-    'parameters' => [],
-    'services' => [
-      'redis.factory' => [
-        'class' => 'Drupal\redis\ClientFactory',
-      ],
-      'cache.backend.redis' => [
-        'class' => 'Drupal\redis\Cache\CacheBackendFactory',
-        'arguments' => ['@redis.factory', '@cache_tags_provider.container', '@serialization.phpserialize'],
-      ],
-      'cache.container' => [
-        'class' => '\Drupal\redis\Cache\PhpRedis',
-        'factory' => ['@cache.backend.redis', 'get'],
-        'arguments' => ['container'],
-      ],
-      'cache_tags_provider.container' => [
-        'class' => 'Drupal\redis\Cache\RedisCacheTagsChecksum',
-        'arguments' => ['@redis.factory'],
-      ],
-      'serialization.phpserialize' => [
-        'class' => 'Drupal\Component\Serialization\PhpSerialize',
-      ],
-    ],
-  ];
-}
-```
-
-The `example.services.yml` file noted above also uses Redis for the lock and flood control systems.
-
-The Redis module can use Redis as a queue backend, but that shouldn't be done on an ephemeral Redis instance as that could result in lost items when the Redis service instance is restarted or fills up.
-If you wish to use Redis for the queue we recommend using a separate persistent Redis instance.
-See the [Redis documentation page](../../add-services/redis.md) for more information.
-
-### Verifying Redis is running
-
-You can verify that Redis is running correctly by connecting to it from an SSH session in your environment.
-After logging in, run
-
-```bash
-echo $PLATFORM_RELATIONSHIPS | base64 --decode | json_pp
-```
-
-to get the list of relationships and find the `host` property for your Redis relationship.
-Then with that value, run
-
-```bash
-redis-cli -h YOUR_REDIS_HOSTNAME info
-```
-
-This should give you a baseline of activity on your Redis installation. There should be very little memory allocated to the Redis cache.
-
-After you push this code, you should run the command and notice that allocated memory will start jumping.
-
-### Clear SQL cache tables
-
-Once you've confirmed that your site is using Redis for caching, you can and should purge any remaining cache data in the MySQL database as it's now just taking up space.
-`TRUNCATE` any table that begins with `cache` *except* for `cache_form`.
-Despite its name `cache_form` isn't part of the cache system proper and thus shouldn't be moved out of SQL.
+Then commit the resulting changes to your `composer.json` 
+and `composer.lock` files.
+
+Alternatively, you can [install the Redis module manually](https://www.drupal.org/project/redis/releases/8.x-1.6), 
+although it is not recommended by Platform.sh or Drupal.
+
+## Configure your Redis service
+
+To configure your Redis service:
+
+1. After [installing the Platform.sh Config Reader library](../../guides/drupal9/deploy/customize.md#install-the-config-reader), 
+   add the following code at the top of your `settings.platformsh.php` file:
```

Comment:
:hammer_and_wrench: This is already above, so probably doesn't need to be here.
```suggestion
1. Add the following code at the top of your `settings.platformsh.php` file:
```

Reply:
:pick: Can include this information in the location
```suggestion
1. Add the following code at the top of your Platform.sh settings file:
```

Reply:
:lady_beetle: This file won't necessarily exist if they haven't used a template, which is one of the two paths noted above. It's possible more context would be necessary here in such cases or a word about it added to the **Before you begin** section.